File: Corte_1/Python/Persona.py
class Persona:
    def __init__(self):
        self.n = ""
        self.e = 0

    def saludar(self):
        print(f"Hola, soy {self.n} y tengo {self.e} años")

    def responder(self):
        print(f"Que tal, yo soy {self.n} y mi edad es {self.e}")

    def mostrar(self):
        print(f"Nombre = {self.n}\nEdad = {self.e}")

# Main
p1 = Persona()
p2 = Persona()

p2.n, p2.e = "Carlos", 19
p1.n, p1.e = "Elena", 23

print("--- Dialogo ---")
p1.saludar()
p2.responder()

print("\n--- Datos ---")
p1.mostrar()
p2.mostrar()

File: Corte_1/Java/persona/src/Persona.java
class datosp {
    public String nombre;
    public int edad;

    public void saludar() {
        System.out.println("Hola, me llamo " + nombre + " y mi edad es " + edad);
    }

    public void responder() {
        System.out.println("Que tal, yo soy " + nombre + " y tengo " + edad + " años");
    }

}

public class Persona {
    public static void main(String[] args) {
        datosp p1 = new datosp();
        datosp p2 = new datosp();

        p2.nombre = "Carlos";
        p2.edad = 19;

        p1.nombre = "Elena";
        p1.edad = 23;

        System.out.println("--- Dialogo ---");
        p1.saludar();
        p2.responder();

        
    }
}